source/models/row.py

```diff
@@ -1,15 +1,19 @@
 import os, json, uuid
-import time
 from outputs.exceptions import *
 from models.index import Index
 
 class Row:
-    def __init__(self, table, value = None):
+    def __init__(self, table, value = {}):
         self.table = table
         self.value = value
```

Comment:
Value should be private

Reply:
why should i be private?
i think Hussin need this variable in the get

Reply:
He should deal with row object. Value is hidden from him

Reply:
I commented on Hussien PR that compare method should be in Row and rename It by has_attribute

Reply:
Read the advantage of encapsulating 
https://www.thoughtco.com/definition-of-encapsulation-958068#:~:text=Advantages%20of%20Encapsulation&text=Encapsulation%20protects%20an%20object%20from,the%20maintenance%20of%20the%20application